frontend/smart-kickers-game/src/components/GameResults.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Button } from '../components/Button';
+import { addPoint, subPoint } from '../apis/goalManipulation.js';
+import './GameResults.css';
+
+function GameResults({ blueScore, whiteScore }) {
+  return (
+    <div>
+      <div className="game-result-container">
+        <div className="game-result-item">
+          <Button
+            onClick={() => {
+              addPoint(2);
+            }}
+          >
+            +
```

Comment:
Why are _blue_ are so priviledged that they get 2 points?

Reply:
It is team ID, not amount of points.

Reply:
🤦

Reply:
But still, then it should be a constant somewhere.

Reply:
So the name of the function should be changed, because it is misleading.

Reply:
Or `addPoint(BLUE_TEAM_ID)`, `subPoint(WHITE_TEAM_ID)` etc.

Reply:
We made an enum with TeamID's as you suggested.